indexer/package.json

```diff
@@ -14,9 +14,7 @@
     "test": "vitest run"
   },
   "dependencies": {
-    "@ethersproject/abi": "^5.8.0",
```

Comment:
Is my understanding correct that we don't need this dependency because you are relying on pieceCids in the event payload? In other words, this PR is addressing https://github.com/filcdn/roadmap/issues/68 as well, correct?